### PR TITLE
misc(docs): fix typos

### PR DESCRIPTION
**Pull request by adeyinkaezra123**
Corrected typos and eliminated redundant words in the functions' docstrings.

### CODE REVIEW

**Comment by adeyinkaezra123** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: inputfiles/idl/fetch.commentmap.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "request-url": "Returns the URL of request as a string.",
     "request-headers": "Returns a Headers object consisting of the headers associated with request. Note that headers added in the network layer by the user agent will not be accounted for in this object, e.g., the \"Host\" header.",
     "request-destination": "Returns the kind of resource requested by request, e.g., \"document\" or \"script\".",
-    "request-referrer": "Returns the referrer of request. Its value can be a same-origin URL if explicitly set in init, the empty string to indicate no referrer, and \"about:client\" when defaulting to the global's default. This is used during fetching to determine the value of the `Referer` header of the request being made.",
+    "request-referrer": "Returns the referrer of request. Its value can be a same-origin URL if explicitly set in init, the empty string to indicate no referrer, and \"about:client\" when defaulting to the global's default. This is used during fetching to determine the value of the `Referrer` header of the request being made.",
```
Comment (saschanaz) [inline inputfiles/idl/fetch.commentmap.json:7]: It's an infamous implementation level typo: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Unfortunately it's way too late to fix it in browsers, so we need to just deal with it.

File: baselines/dom.generated.d.ts [hunk 2]
```diff
@@ -6327,7 +6327,7 @@ declare var DOMRectReadOnly: {
  */
 interface DOMStringList {
     /**
-     * Returns the number of strings in strings.
+     * Returns the number of strings in a list of DOMString (strings).
```
Comment (saschanaz) [inline baselines/dom.generated.d.ts:6330]: This is not really a typo but a rephrase, can you split this in a separate PR at least?

**Review by saschanaz — COMMENTED** [review]
The commentmap.json and apiDescriptions are generated by specifications, but for a long time it has been stalled. I can merge this when the feedbacks are solved.

File: inputfiles/idl/fetch.commentmap.json (hunk 1 above)
Comment (adeyinkaezra123) [reply inputfiles/idl/fetch.commentmap.json:7]: Oh, wow 😂

File: baselines/dom.generated.d.ts (hunk 2 above)
Comment (adeyinkaezra123) [reply baselines/dom.generated.d.ts:6330]: Oh, okay, would close this PR and raise a new one with a cleaner commit tree
